src/middleware/createRouter.ts

```diff
@@ -51,6 +51,8 @@ export const createRouter: RoutingManager = (props = {}) => {
     };
 
     return {
+      $$type: 'aism.router',
```

Comment:
I've added `$$type` to the middleware to distinguish middlwares.
Not sure about the prefix `aism.` though.

Reply:
I think `ais` is good enough